Packs/ApiModules/Scripts/NGINXApiModule/NGINXApiModule.py

```diff
@@ -299,6 +303,9 @@ def run_long_running(params: Dict = None, is_test: bool = False):
 
         else:
             nginx_process = start_nginx_server(nginx_port, params)
+            # let nginx startup
+            time.sleep(0.5)
```

Comment:
Let's move this into the `test_nginx_web_server` server and also add a polling loop. So the function `test_nginx_web_server` will try the http request 5 times max each with a 0.5 delay. And only raise the exception after the 5th time.

Reply:
Done